config/webhook/manifests.yaml

```diff
@@ -2,18 +2,18 @@
 apiVersion: admissionregistration.k8s.io/v1
 kind: ValidatingWebhookConfiguration
 metadata:
-  name: ptpconfig-validating-webhook-configuration
+  creationTimestamp: null
+  name: validating-webhook-configuration
```

Comment:
why this is not ptpconfig-validating-webhook-configuration

Reply:
`make manifests` will always overwrite this back to `validating-webhook-configuration`, that's why the patch `patches/patch_webhook_configuration.yaml` is introduced to update this to `ptpconfig-validating-webhook-configuration`.